ros/processor/garbage_collector.py

```diff
@@ -0,0 +1,28 @@
+from ros.lib.app import app, db
+from ros.lib.models import PerformanceProfile
+from datetime import datetime, timedelta
+from ros.lib.config import GARBAGE_COLLECTION_INTERVAL, DAYS_UNTIL_STALE, get_logger
+import time
+
+LOG = get_logger(__name__)
+
+
+class GarbageCollector():
+    def run(self):
+        while True:
+            self.remove_outdated_data()
+
+    def remove_outdated_data(self):
+        with app.app_context():
+            results = db.session.query(PerformanceProfile).filter(
+                      PerformanceProfile.report_date < datetime.today() - timedelta(days=DAYS_UNTIL_STALE)).delete()
```

Comment:
```suggestion
                      PerformanceProfile.report_date < datetime.now(timezone.utc) - timedelta(days=DAYS_UNTIL_STALE)).delete()
```

@patilsuraj767 - As `report_date` stored in UTC, do we need to consider `datetime.now(timezone.utc)1 over `datetime.today()`?